### PR TITLE
[Solution] PAT-B1003.我要通过！python code

### PR DESCRIPTION
**Pull request by 594Surefour**
Signed-off-by: paul_Lee <1337998606@qq.com>

---
name: PR
about: Solutions of PAT-B1003.我要通过！
title: "[Solution] PAT-B1003.我要通过！"
labels: status/pending
assignees: @594Surefour 

---

## 参与必读！！！

* **贡献指南：** https://os-edu.github.io/contribution/add_contributing_guide/

## 添加题目解法

* [相关链接（建议）]：Issue #190 
* [题目信息（必选）]：https://pintia.cn/problem-sets/994805260223102976/problems/994805323154440192
* [题解思路（可选）]：

### CODE REVIEW

**Comment by 594Surefour** [comment]
> Signed-off-by: paul_Lee [1337998606@qq.com](mailto:1337998606@qq.com)
> 
> name: PR
> about: Solutions of PAT-B1003.我要通过！
> title: "[Solution] PAT-B1003.我要通过！"
> labels: status/pending
> assignees: @594Surefour
> 
> ## 参与必读！！！
> * **贡献指南：** https://os-edu.github.io/contribution/add_contributing_guide/
> 
> ## 添加题目解法
> * [相关链接（建议）]：Issue [[Solution] 提交 PAT-B1003.我要通过! C++ 解法 #207](https://github.com/OS-EDU/KO--CSP/pull/207)
> * [题目信息（必选）]：https://pintia.cn/problem-sets/994805260223102976/problems/994805323154440192
> * [题解思路（可选）]：

**Comment by 594Surefour** [comment]
> Signed-off-by: paul_Lee [1337998606@qq.com](mailto:1337998606@qq.com)
> 
> name: PR
> about: Solutions of PAT-B1003.我要通过！
> title: "[Solution] PAT-B1003.我要通过！"
> labels: status/pending
> assignees: @594Surefour
> 
> ## 参与必读！！！
> * **贡献指南：** https://os-edu.github.io/contribution/add_contributing_guide/
> 
> ## 添加题目解法
> * [相关链接（建议）]：Issue [[Solution] 提交 PAT-B1003.我要通过! C++ 解法 #207](https://github.com/OS-EDU/KO--CSP/pull/207)
> * [题目信息（必选）]：https://pintia.cn/problem-sets/994805260223102976/problems/994805323154440192
> * [题解思路（可选）]：

**Comment by 594Surefour** [comment]
> Signed-off-by: paul_Lee [1337998606@qq.com](mailto:1337998606@qq.com)
> 
> name: PR
> about: Solutions of PAT-B1003.我要通过！
> title: "[Solution] PAT-B1003.我要通过！"
> labels: status/pending
> assignees: @594Surefour
> 
> ## 参与必读！！！
> * **贡献指南：** https://os-edu.github.io/contribution/add_contributing_guide/
> 
> ## 添加题目解法
> * [相关链接（建议）]：Issue [[Solution] 提交 PAT-B1003.我要通过! C++ 解法 #207](https://github.com/OS-EDU/KO--CSP/pull/207)
> * [题目信息（必选）]：https://pintia.cn/problem-sets/994805260223102976/problems/994805323154440192
> * [题解思路（可选）]：

File: PAT/pythonSrc/PAT-B1003.我要通过！.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            index_P = i
+            count_P += 1
+        elif s[i] == 'T':
+            index_T = i
+            count_T += 1
+        elif s[i] == 'A': 
+            sum_A += 1
+    if count_P == 1 and count_T == 1 and index_P < index_T and sum_A and sum_A == l-2:
+        left_A = index_P
+        mid_A = index_T - index_P - 1
```
Comment (QuakeWang) [inline PAT/pythonSrc/PAT-B1003.我要通过！.py:20]: 为了规范化代码，需要解决一下这个：More than one statement on a single line (multiple-statements)

**Review by QuakeWang — COMMENTED** [review]
hi, 可以增加适当的注释，提高阅读性

File: PAT/pythonSrc/PAT-B1003.我要通过！.py (hunk 1 above)
Comment (594Surefour) [reply PAT/pythonSrc/PAT-B1003.我要通过！.py:20]: 这句话 我pr的时候看到了，不知道为啥有问题嘞🤔🤔 代码能正常运行，答案也是正确的

Comment (QuakeWang) [reply PAT/pythonSrc/PAT-B1003.我要通过！.py]: > 这句话 我pr的时候看到了，不知道为啥有问题嘞🤔🤔 代码能正常运行，答案也是正确的

可以使用代码格式化插件，会自动规范代码的~~

**Comment by 594Surefour** [comment]
> hi, 可以增加适当的注释，提高阅读性

已添加相应注释～！

**Review by QuakeWang — APPROVED** [review]
/approve

**Pull request merged**